### PR TITLE
Merge changes from finalized 2.31 build

### PR DESCRIPTION
**Pull request by mokagio**
- [x] `versionName` dropping the `rc` suffix and `versionCode` bump in `build.gradle`
- [x] `strings.xml` updates (none were expected given there have been no changes in the source, but we got one that I can only assume was a late arrival or a community tweak)
- [x] `release_notes.xml` and `changelogs/*.txt` updates

@spencertransier @oguzkocer thank you in advance for reviewing this, you were suggested by GitHub 😅

### CODE REVIEW

File: fastlane/metadata/android/release_notes.xml [hunk 1]
```diff
@@ -1,4 +1,85 @@
...
+*修復了應用程式啟動時因筆記量過大而沒有反應的問題。
+
+</zh-TW>
 <en-US>
-2.30:
-• Fixed a crash when syncing large notes
+2.31:
+* Fixed the app from becoming unresponsive at launch with a large amount of notes.
+
 </en-US>
```
Comment (mokagio) [inline fastlane/metadata/android/release_notes.xml:85]: Out of scope given the state of this project, but I wonder whether we should add this to the `.gitignore` given it's build from the individual changelogs 🤔

Comment (spencertransier) [reply fastlane/metadata/android/release_notes.xml:85]: We can probably remove this file and the automation that creates it once we use `upload_to_play_store` to automatically upload the release notes to the Play Store tracks. 

See the "Changelogs (What's new)" section in https://docs.fastlane.tools/actions/upload_to_play_store

**Review by spencertransier — APPROVED** [review]
Looks great!

File: fastlane/metadata/android/release_notes.xml (hunk 1 above)
Comment (mokagio) [reply fastlane/metadata/android/release_notes.xml:85]: Great idea. Now... how do we remember to do this on the next release given this project is in maintenance mode 🤔

Comment (mokagio) [reply fastlane/metadata/android/release_notes.xml:85]: Maybe we should leave a note in the release scenario to implement this change before starting the next code freeze.

Comment (mokagio) [reply fastlane/metadata/android/release_notes.xml:85]: I opened a PR for that in one of our internal repos.

**Pull request merged**